actors/common/src/main/java/org/sunbird/learner/actors/health/HealthActor.java

```diff
@@ -181,13 +181,13 @@ private void checkAllComponentHealth() {
             PropertiesCache.getInstance().getProperty(JsonKey.EKSTEP_AUTHORIZATION));
         headers.put("Content_Type", "application/json; charset=utf-8");
       }
-      String ekStepBaseUrl = System.getenv(JsonKey.EKSTEP_BASE_URL);
-      if (StringUtils.isBlank(ekStepBaseUrl)) {
-        ekStepBaseUrl = PropertiesCache.getInstance().getProperty(JsonKey.EKSTEP_BASE_URL);
+      String searchBaseUrl = System.getenv(JsonKey.SEARCH_SERVICE_API_BASE_URL);
```

Comment:
ProjectUtil has method to get config value directly from either env or properties file.